tests/unit/htdocs/Frameworks/art/FunctionsIniTest.php

```
@@ -0,0 +1,301 @@
+    }
+
+    // ---------------------------------------------------------------
+    // Helper
+    // ---------------------------------------------------------------
+
+    private function createMockModule(string $dirname): object
+    {
+        return new class($dirname) {
+            private string $dirname;
+            public function __construct(string $dirname) { $this->dirname = $dirname; }
+            public function getVar(string $key, string $format = 'e')
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$format'. (undefined)

(UnusedFormalParameter)